pkg/firewall/firewall.go

```diff
@@ -172,7 +172,7 @@ func (soakp *stakeOrActiveKeepPolicy) validateActiveKeepMembership(
 	lastIndex := new(big.Int).Sub(keepCount, one)
```

Comment:
Can we add some additional logs we used for debugging in https://github.com/keep-network/keep-ecdsa/commit/6273da259611182be253200ca773180bcded549d and https://github.com/keep-network/keep-ecdsa/commit/b4a01bca8bdc90db25f02006624a22c47ea18214?

Reply:
I suggest we discuss it for the backport PR and add no more logging for this fix.